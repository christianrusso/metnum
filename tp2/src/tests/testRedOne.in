../data/ImagenesCarasRed/ 28 23 1 5 3 
s1/ 1 2 3 4 5
5
../data/ImagenesCarasRed/s1/10.pgm 1
../data/ImagenesCarasRed/s2/10.pgm 2
../data/ImagenesCarasRed/s3/10.pgm 3
../data/ImagenesCarasRed/s4/10.pgm 4
../data/ImagenesCarasRed/s5/10.pgm 5