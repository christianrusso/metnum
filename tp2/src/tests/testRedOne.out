373.3930788
290.4364266
235.3437204
